modules/agar/src/main/ts/ephox/agar/api/FocusTools.ts

```diff
@@ -106,6 +106,7 @@ const sSetActiveValue = <T>(doc: SugarElement<Document | ShadowRoot>, newValue:
 export {
   setActiveValue,
   setFocus,
+  getFocused,
```

Comment:
Note to review: Exporting this now

Reply:
Why not just use `Focus.active` directly instead? That's the reason I didn't export this, as I didn't want 2 ways to do the same thing.

Reply:
We decided in a call that we'll keep this, as it keeps the error message consistent and will allow us to assert types for tests.